symbulate/tests/test_distributions.py

```diff
@@ -17,18 +17,460 @@ def test_p_one(self):
     def test_sum(self):
         X = RV(Bernoulli(.4) ** 5)
         sims = X.apply(sum).sim(Nsim)
+        key_list = [keys for keys in sims.tabulate()]
         obs = list(sims.tabulate().values())
-        exp = Nsim * stats.binom(n=5, p=.4).pmf(range(6))
+        exp = [Nsim * stats.binom(n = 5, p = .4).pmf(k) for k in key_list]
+        pval = stats.chisquare(obs, exp).pvalue
+        self.assertTrue(pval > 0.01)
+
+    def test_Bernoulli_Binomial_n_1(self):
+        exp, obs, new_key = [], [], []    
+        X = RV(Bernoulli(.4))
+        sims = X.sim(Nsim)
+        key_list = list(sims.tabulate().keys())
```

Comment:
It shouldn't be necessary to convert the keys to a list. You should be able to iterate over the dict_keys object directly.